src/pymatgen/io/phonopy.py

```diff
@@ -329,7 +329,7 @@ def get_phonon_dos_from_fc(
 
     phonon.run_projected_dos(freq_min=freq_min, freq_max=freq_max, freq_pitch=freq_pitch)
 
-    dos_raw = phonon.projected_dos.get_partial_dos()
+    dos_raw = (phonon.projected_dos.frequency_points, phonon.projected_dos.projected_dos)
```

Comment:
I believe this is the right replacement, but always better to double check, here is the source code:
```python
    def get_partial_dos(self):
        """Return partial DOS.

        Returns
        -------
        tuple
            frequency_points: Sampling frequencies
            projected_dos: [atom_index, frequency_points_index]

        """
        warnings.warn(
            "ProjectedDos.get_partial_dos() is deprecated. "
            "Use frequency_points and projected_dos attributes instead.",
            DeprecationWarning,
            stacklevel=2,
        )
        return self._frequency_points, self._projected_dos
```